src/common/Select/Select.tsx

```diff
@@ -176,14 +190,14 @@ export const Select: SelectComponent<SelectProps> = ({
       <Block ref={rootRef} name="select" mod={{ size }} style={style} tabIndex={tabIndex} onKeyDown={handleKeyboard}>
         <Dropdown.Trigger
           ref={dropdown}
-          style={{ maxHeight: 280, overflow: 'auto' }}
+          style={{ maxHeight: 280, overflow: 'auto', ...dropdownStyle }}
```

Comment:
maxHeight and overflow could should probably get moved to css, dropdown style is getting passed in to the component so i guess thats fine since its dynamic